textalyze.js

```diff
@@ -1,7 +1,16 @@
+/**
+ * Given an input String, returns true if it's a valid string false otherwise
+ * @param {String} string - The String to be sanitized
+ * @returns {Boolean} The String sanitized
+ */
+function validateString(string) {
```

Comment:
In JavaScript, I think a function like this would normally be called `isValidString` (like `isNaN`, `isInteger`, etc.).

I'd also just call it `isString` since it's not validating anything — it's just telling you whether the object is a string.

Reply:
Oh, and now I see I made a mistake!  I assumed `validateString` would return `true` is the object is a string, but it's the other way around (it returns `false` if the object is a string).

That mistake I just made is a consequence of how the function is named, so I'd _definitely_ call it `isString` and have it return `true` if the object is a string.  Then later you can check `if (!isString(object)) { ... }` and it will be perfectly clear what's happening.